content/reference/plugins/github_actions/index.md

```diff
@@ -0,0 +1,189 @@
+- `runs_status_name`: The name of the run's final status (e.g. `Successful`, `AssertionsSuccessful`, `AssertionsFailed`, etc.).
+- `runs_status_code`: The code of the run's final status.
+- `run_assertions`: The results of the run's assertions, as a JSON array.
+
+### Logs
+
+Every few seconds, the action logs to the GitHub Action console a summary of the run's current status. When the run ends, the Action logs the status of the run and the results of any assertions. Here's a very short duration example:
+
+{{< img src="reference_logs.png" alt="A run's logs in the GitHub Actions console" >}}
+
+### Cancellation
+
+When the Action starts, it registers a post-execution, clean-up task in the workflow. If the Action fails or gets cancelled by a user, and if the simulation is still running on Gatling Enterprise, this clean-up task will attempt to cancel the execution on Gatling Enterprise.
+
+## Sample use cases
+
+### Build and run simulation
+
+This workflow is defined in the GitHub repository which contains your Gatling simulation script built with one of our build tools plugins. In this example, every time the code on the `main` branch gets updated, we:
+
+- build, package, and upload to Gatling Enterprise the current version of the simulation script
+- run the updated simulation on Gatling Enterprise 
+
+Feel free to use different trigger events or to configure the other inputs and outputs for the Action as documented above, according to your own use case.
+
+```yaml
+name: Run Gatling Enterprise Simulation
+
+# Execute the workflow on each push to the main branch
+on:
+  push:
+    branches:
+      - main
+
+# Here we use concurrency to cancel previous executions if they are still
+# ongoing. Useful to avoid running the same simulation several times
+# simultaneously if we push several code changes in a short time.
+# See https://docs.github.com/actions/using-jobs/using-concurrency.
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+# The GATLING_ENTERPRISE_API_TOKEN environment variable is recognized by both
+# the Gatling Maven plugin and the gatling/enterprise-action Action
+env:
+  SIMULATION_ID: 'xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx'
+  GATLING_ENTERPRISE_API_TOKEN: ${{ secrets.GATLING_ENTERPRISE_API_TOKEN }}
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    steps:
+      # Check out your GitHub repository.
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      # Set up Java and the build tools (Maven, Gradle, and SBT).
+      # You can configure other versions of the JDK, as long as they are
+      # supported by your version of Gatling and by your build tool.
+      # See https://github.com/actions/setup-java/blob/main/README.md for options.
+      - name: Setup JDK
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'zulu'
+          java-version: '17'
+          cache: 'maven' # or 'gradle' if you use Gradle, 'sbt' if you use SBT
```

Comment:
Another one I just learned about: `setup-java` v3 can automatically set up caches for the build tools' local dependency caches (see [the documentation here](https://github.com/actions/setup-java#caching-packages-dependencies)). It makes some assumptions for the cache key, but seems to makes sense at least for typical/simple projects.